en/05_Contributing/02_Build_Tooling.md

```diff
@@ -18,148 +18,133 @@ The [NodeJS](https://nodejs.org) JavaScript runtime is the foundation of our cli
-for easier debugging. It is important to note that this should not be used for pushing up changes,
-and you should run `yarn run build` after you're done.
+The same as `yarn dev`, except it will automatically rebuild whenever you change a `.js` or `.scss` file.
+This is useful for when you are rapidly making lots of small changes.
 
-```
-$ yarn run watch
+```sh
+yarn build
 ```
 
-### css
-
-```
-$ yarn run css
-```
+Runs [Webpack](https://webpack.github.io/) to builds the core JavaScript and CSS files in production mode.
+You will need to run this script before committing your changes to git.
 
-Compiles all of the `.scss` files into minified `.css` files.
+### build JavaScript or CSS separately
 
-Run this script with `-- --watch` to automatically rebuild on file changes.
-The first `--` separator is required to separate arguments from NPM's own ones.
+If you are only working on JavaScript or only working on CSS you might want to only build what you're working on. You can do this by adding `WEBPACK_CHILD=css` or `WEBPACK_CHILD=js` before the relevant yarn command, for example:
 
+```sh
+WEBPACK_CHILD=css yarn dev
 ```
-$ yarn run css -- --watch
-```
+
+The `css` or `js` portion of this is defined in the `webpack.config.js` file. Some modules may also include other configured components that can be built independently as well.
 
 ### lint
 
-```
-$ yarn run lint
+```sh
+yarn lint
 ```
 
-Run linters (`eslint` and `sass-lint`) linters to enforce
-our [JavaScript](/contributing/javascript_coding_conventions) and 
+Run linters (`eslint` and `sass-lint`) to enforce
+our [JavaScript](/contributing/javascript_coding_conventions) and
 [CSS](/contributing/css_coding_conventions) coding conventions.
 
 ### test
 
-```
-$ yarn run test
+```sh
+yarn test
 ```
 
 Runs the JavaScript unit tests.
 
 ### coverage
 
-```
-$ yarn run coverage
+```sh
+yarn coverage
 ```
 
 Generates a coverage report for the JavaScript unit tests. The report is generated
 in the `coverage` directory.
 
-## Requiring SilverStripe ES6 Modules in your own CMS customisation
+## Requiring Silverstripe JavaScript modules in your own CMS customisation
 
-SilverStripe creates bundles which contain many dependencies you might also
+Silverstripe creates bundles which contain many dependencies you might also
 want to use in your own CMS customisation (e.g. `react`).
-You might also need some of SilverStripe's own ES6 modules (e.g. `components/FormBuilder`).
+You might also need some of SilverStripe's own JavaScript ECMA modules (e.g. `components/FormBuilder`).
 
-To avoid double including these in your own generated bundles,
-we have exposed many libraries as [Webpack externals](https://webpack.github.io/docs/library-and-externals.html).
+To avoid transpiling these into your own generated bundles,
+we have exposed many libraries as [Webpack externals](https://webpack.js.org/configuration/externals/).
```

Comment:
```suggestion
To avoid transpiling these into your own generated bundles, we have exposed many libraries as [Webpack externals](https://webpack.js.org/configuration/externals/).
```

Reply:
I don't understand the need for this change, but it won't change anything so I'll do it lol

Reply:
https://github.com/silverstripe/developer-docs/pull/130#discussion_r1054846663